### PR TITLE
Aryanmajeed/issue19

### LINKED ISSUE #19: Create settings components
**Is your feature request related to a problem? Please describe.**
N/A

**Describe how the solution should behave**
The user should be able to edit his info in this component and it should be responsive.

**Screenshots of the design/if any**
image

**Additional context**
Use formik or react hook form for validation.

### PR DESCRIPTION
**Pull request by aryankarim**
## What?
This PR contains the Account Setting route where users can change their profile data. 
## Why?
This feature gives the ability to users to express themselves by adding hobbies, bio, etc.
## How to use?
go to `http://localhost:3000/setting` route
## Screenshots 
screencapture-localhost-3000-setting-2021-10-17-19_27_56
screencapture-localhost-3000-ar-setting-2021-10-17-19_28_35

### CODE REVIEW

**Comment by aryankarim** [comment]
PS: save button is also added! Also, I will add the localization tomorrow.

File: pages/setting/index.js [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+            </WrapItem>
+            <WrapItem alignItems="center">
+              <Button
+                rounded="5px"
+                backgroundColor="lightPurple"
+                color="white"
+                _hover={{ bg: "darkPurple" }}
+              >
+                {t("uploadNewPhoto")}
+              </Button>
```
Comment (HaiderAltufaily) [inline pages/setting/index.js]: Great work,
It would be awesome if you can change this button to an input to upload images

**Review by NooraYaqub44 — APPROVED** [review]

**Comment by NooraYaqub44** [comment]
great job

File: pages/setting/index.js (hunk 1 above)
Comment (aryankarim) [reply pages/setting/index.js]: Sure I'm on it! 🐦

**Review by HaiderAltufaily — APPROVED** [review]

**Comment by aryankarim** [comment]
@MuhannedNoman We need final review! 🙌

**Comment by MuhannedNoman** [comment]
The mobile view looks add
image
Also, can you add place holder to text fields like the textarea they look weird empty!

**Comment by aryankarim** [comment]
> The mobile view looks add image Also, can you add place holder to text fields like the textarea they look weird empty!

Sure I'll fix those right now!

**Comment by aryankarim** [comment]
Done! @MuhannedNoman 
screencapture-localhost-3000-setting-2021-10-18-19_00_18

**Comment by aryankarim** [comment]
@MuhannedNoman This is better. right! 👨‍💻👨‍💻

**Review by MuhannedNoman — APPROVED** [review]
LGTM, let's put a placeholder image instead of the guy image.

**Pull request merged**